### PR TITLE
[201911]: add show bgp neigh/network support for multi asic

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>

#### What I did
This is port of the PR https://github.com/Azure/sonic-utilities/pull/1574 in 201911 branch

This PR is to add support for the commands show ip bgp neighbor and show ip bgp network
Add unit tests for these commands

How I did it
Add change to get the bgp neighbor and network information from each namespace

How to verify it
Check the command works on single and multi asic platforms

UT  results
============================= test session starts ==============================
platform linux2 -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /sonic/src/sonic-utilities, inifile: pytest.ini
plugins: cov-2.4.0
collected 314 items

sonic-utilities-tests/acl_config_test.py ............
sonic-utilities-tests/acl_loader_test.py .....................
sonic-utilities-tests/aclshow_test.py ..........
sonic-utilities-tests/counterpoll_test.py ...
sonic-utilities-tests/crm_test.py ..................................................
sonic-utilities-tests/db_migrator_test.py ..............................
sonic-utilities-tests/drops_group_test.py .......
sonic-utilities-tests/feature_test.py ...............
sonic-utilities-tests/filter_fdb_entries_test.py ..........
sonic-utilities-tests/intfstat_test.py ........
sonic-utilities-tests/intfutil_test.py .........
sonic-utilities-tests/ip_show_routes_multi_asic_test.py ...............
sonic-utilities-tests/ip_show_routes_test.py ......
sonic-utilities-tests/multi_asic_intfutil_test.py ..........
sonic-utilities-tests/pfcstat_test.py .......
sonic-utilities-tests/pfcwd_test.py ............
sonic-utilities-tests/port2alias_test.py ....
sonic-utilities-tests/portstat_test.py ..................
sonic-utilities-tests/psu_test.py ...
sonic-utilities-tests/sfp_test.py .........
sonic-utilities-tests/show_bgp_neighbor_test.py ............
sonic-utilities-tests/show_bgp_network_test.py .................
sonic-utilities-tests/show_ip_bgp_commands_test.py ...........
sonic-utilities-tests/show_ip_int_test.py ..........
sonic-utilities-tests/sku_create_test.py .....

========================= 314 passed in 65.28 seconds ==========================
make[1]: Leaving directory '/sonic/src/sonic-utilities/deb_dist/sonic-utilities-1.2'
 fakeroot debian/rules binary

### CODE REVIEW

File: utilities_common/bgp_util.py [hunk 1]
```diff
@@ -10,6 +10,32 @@
...
+    bgp_session = config_db.get_entry(
+        multi_asic.BGP_INTERNAL_NEIGH_CFG_DB_TABLE, neighbor_ip)
+    if bgp_session:
+        return True
+    return False
+
+
 
 def is_ipv4_address(ip_address):
     """
```
Comment (gechiang) [inline utilities_common/bgp_util.py:41]: On line 192:
ctx.fail("Unable to get summary from bgp".format(bgp_instance_id))
There was a fix in Master for the above line which seems to be missing in your porting:
ctx.fail("Unable to get summary from bgp {}".format(bgp_instance_id))

**Review by gechiang — COMMENTED** [review]
There were changes in config_db.json under asic0 and asic1 in the master branch change set.
Are these 2 file changes not required for the 201911 branch?

File: utilities_common/bgp_util.py (hunk 1 above)
Comment (arlakshm) [reply utilities_common/bgp_util.py:41]: I fixed this in the latest commit

**Comment by arlakshm** [comment]
> There were changes in config_db.json under asic0 and asic1 in the master branch change set.
> Are these 2 file changes not required for the 201911 branch?

@gechiang, yes the config_db.json changes are not required. The bgp_neighbor configuration is already present.

**Review by gechiang — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Pull request merged**